docs/usage/contributing.rst

```diff
@@ -0,0 +1,106 @@
+Contributing to PlaidML
+#######################
+
+We welcome contributions to PlaidML from anyone. This document contains:
+
+* Guidelines for creating successful PRs
+
+* Outlines the contribution process
+
+* Lists general areas for contribution
+
+* Provides resources and context to ease development, where relevant and
+
+  available
+
+Before starting any work, please ensure you are able to build and test PlaidML.
+
+
+Guidelines
+**********
+
+* Create unit tests for new features and bug fixes. Integration tests are
+
+  required for larger features.
+
+* Pre-commit linters will be available soon.
```

Comment:
Pre-commit linters are already available. But in general, we may need to rewrite this.